README.md

```diff
@@ -1,2 +1,4 @@
 # ambianic-subscriptions
 Repo for ambianic premium subscription management
+
+This repo hosts [Netlify functions](https://www.netlify.com/products/functions/) that handle Ambianic premium user subscriptions. This is code that cannot run in the browser PWA, because it needs access keys to subscription and payment gateway services (Stripe) shared between all app users. If the code runs in the PWA, users will be able to see and manipulate each other's premium subscription data.
```

Comment:
@vickywane can you please insert a high level architecture diagram that illustrates roughly the intended interactions between ambianic PWA, ambianic serverless functions (this repo) hosted by netlify(aws), Auth0, stripe and ambianic edge.

Reply:
Here is a template to start from: https://app.diagrams.net/#G1zj8sg3aHGFKSvCbhcQ5BHEmWU4GAC-6X